en/snap/preparation.md

```diff
@@ -1,23 +1,25 @@
 # Preparation Before Integrating with Snap
 <hr>
+This section will explain what preparations needed to start Snap integration.
```

Comment:
This section explains what preparations are required to start with Snap integration.